index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <title>My-Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div id="nav-container">
+                <div class="shapes">
+                    <img src="assets/shapes/image_geometry_1.jpg" alt="">   
+                </div>
+                <div class="ic_icon">
+                    <img src="assets/shapes/ic_menu.jpg" alt="">
+                </div>
+            </div>
+        </nav>
+        <div class="headline-section">
+            <div class="header-text">
+                <p>Charlie</p>
+                <p>Giorgania</p>
+            </div>
+            <div class="header-description">
+                <p>Hello! I am a software developer! I can help
+                    you build a product, feature or website. Take a look at my work.
+                </p>
+                <p>If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="headline-button">
+                <p>Start collaboration</p>
+            </div>
+        </div>
```

Comment:
- [ ] Kindly make sure the the container of the section is a `section` tag as it represents a generic standalone section of the document. I recommend to firstly put the above code out of the `header` tag then wrap the code in a `section` tag. Below you can see an example of good semantic tag.
```HTML
    <header>
        <!-- Code for the header here -->
    </header>

    <main>
        <section>
            <!-- Code for the 1st section her -->
        </section>

        <section>
            <!-- Code for the 2nd section her -->
        </section>
        ...
    </main>

    <footer>
        <!-- Code for the footer here -->
    </footer>
```